MediaModuleInstaller.php

```diff
@@ -140,6 +140,13 @@ public function upgrade($oldversion)
                 $this->schemaTool->update(['Cmfcmf\Module\MediaModule\Entity\Watermark\TextWatermarkEntity']);
                 $this->container->get('doctrine.dbal.connection')->executeUpdate("UPDATE `cmfcmfmedia_watermarks` SET `fontColor`='#000000ff',`backgroundColor`='#00000000' WHERE `discr`='text'");
 
+                $this->schemaTool->update([
+                    'Cmfcmf\Module\MediaModule\Entity\Collection\CollectionEntity',
+                    'Cmfcmf\Module\MediaModule\Entity\Collection\CollectionCategoryAssignmentEntity',
+                    'Cmfcmf\Module\MediaModule\Entity\Media\AbstractMediaEntity',
+                    'Cmfcmf\Module\MediaModule\Entity\Media\MediaCategoryAssignmentEntity',
+                ]);
+
```

Comment:
Shouldn't also registries be created during the install process? Otherwise the admin must do it manually afterwards.